src/pages/components/toggle-icon-button.mdx

```diff
@@ -0,0 +1,151 @@
+---
+title: Toggle icon button
+description: >
+  A Toggle Icon Button (like a Toggle Button) lets users turn something on
+  and off like a light switch. It is used for repeated or persistent actions
+  on a page and lacks a visible label to simplify the UI.
+package: '@zendeskgarden/react-buttons'
+components:
+  - buttons/src/elements/ToggleIconButton.tsx
+---
+
+<Usage>
+  <Use>
+    <ul>
+      <li>To see or compare the results of a settings change</li>
+      <li>To activate a mode (such as "dark mode") which takes immediate effect</li>
+      <li>To simplify the appearance of repeated or persistent actions on a page</li>
+      <li>To enable action in a toolbar</li>
+    </ul>
+  </Use>
+  <Misuse>
+    <ul>
+      <li>
+        <Markdown>
+          To let users select from a list of settings, use [Checkboxes](/components/checkbox)
+          instead
+        </Markdown>
+      </li>
+      <li>
+        <Markdown>To include an icon and a label, use a [Tile](/components/tiles)</Markdown>
+      </li>
+    </ul>
+  </Misuse>
+</Usage>
+
+## How to use it
+
+### Default
+
+The typical usage of a Toggle Icon Button component.
+
+import ToggleIconButtonDefault from '../../examples/button/ToggleIconButtonDefault.tsx';
+import ToggleIconButtonDefaultCode from '!!raw-loader!../../examples/button/ToggleIconButtonDefault.tsx';
+
+<CodeExample code={ToggleIconButtonDefaultCode}>
+  <ToggleIconButtonDefault />
+</CodeExample>
+
+### Disabled
+
+A disabled Toggle Icon Button prevents user interaction. It doesn’t appear in
+the tab order, can’t receive focus, and may not be read aloud by a
+screenreader.
+
+import ToggleIconButtonDisabled from '../../examples/button/ToggleIconButtonDisabled.tsx';
+import ToggleIconButtonDisabledCode from '!!raw-loader!../../examples/button/ToggleIconButtonDisabled.tsx';
+
+<CodeExample code={ToggleIconButtonDisabledCode}>
+  <ToggleIconButtonDisabled />
+</CodeExample>
+
+### Icon
+
+A Toggle Icon Button may be structured to switch icons when pressed.
+
+import ToggleIconButtonIcons from '../../examples/button/ToggleIconButtonIcons.tsx';
+import ToggleIconButtonIconsCode from '!!raw-loader!../../examples/button/ToggleIconButtonIcons.tsx';
+
+<CodeExample code={ToggleIconButtonIconsCode}>
+  <ToggleIconButtonIcons />
+</CodeExample>
```

Comment:
@ginnywood I went rogue and added this section without content review, etc. Will wait for you to get the heading and description done just right. :pray:

Reply:
I also went rogue and added a section to the Google Doc without content review. Mentioned you and Allison there.

Reply:
Just looked over the content now and it looks good to me!